app/pods/publications/route.js

```diff
@@ -30,12 +30,18 @@ export default class PublicationsRoute extends Route {
       'page[size]': 100,
       sort: 'priority',
     });
+    const mandateePromise = await this.store.query('mandatee', {
+      include: 'person',
+      'page[size]': 100,
+      sort: 'person.last-name',
+    });
```

Comment:
Je kan de mandatees niet als een codelist beschouwen zoals de publication-status of document-types. Op development zijn er maar een 10-tal, maar op productie gaat het aantal mandatarissen richting 1000. Je kan deze dus best niet pre-loaden hier.

Reply:
Kunnen we hier de actieve ministers ophalen (endDate > today) ? dat zijn er max 11 (meer ministers kan dacht ik niet)
En bij zoeken op naam kunnen we dan in Alle ministers zoeken, in geval dat ze een publicatie willen toewijzen op een afgetrede minister